src/app/clusters/scenes/SceneTableImpl.cpp

```diff
@@ -267,7 +267,8 @@ struct FabricSceneData : public PersistentData<kPersistentFabricBufferMax>
     /// @param storage [in] Persistent Storage Delegate, required to delete scenes if the number of scenes in storage is greater
     /// than the maximum allowed
     /// @param deleted_scenes_count [out] uint8_t letting the caller (in this case the load method) know how many scenes were
-    /// deleted so it can adjust the fabric and global scene count accordingly
+    /// deleted so it can adjust the fabric and global scene count accordingly. Upon failure of Deserialize, this value will return
```

Comment:
```suggestion
    /// deleted so it can adjust the fabric and global scene count accordingly. Even if Deserialize fails, this value will return
```
because what's there now sounds like this value is _only_ set on failure.